profiles/mojoshader_profile_glsl.c

```diff
@@ -855,12 +882,20 @@ void emit_GLSL_attribute(Context *ctx, RegisterType regtype, int regnum,
                          int flags)
 {
     // !!! FIXME: this function doesn't deal with write masks at all yet!
+    const char *qualifier_in = "varying";
```

Comment:
Should this be `attribute`?

Reply:
Sorta - varying for fragment, attribute for vertex. It's indeed a silly mistake, fixed.